kde-apps/kalzium/kalzium-5.9999.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_BRANCH="frameworks"
+KDE_HANDBOOK="forceoptional"
+inherit kde5 flag-o-matic
+
+DESCRIPTION="Periodic table of the elements"
+HOMEPAGE="https://www.kde.org/applications/education/kalzium
+https://edu.kde.org/kalzium"
+KEYWORDS=""
+IUSE="editor solver"
+
+DEPEND="
+	$(add_frameworks_dep karchive)
+	$(add_frameworks_dep kcompletion)
+	$(add_frameworks_dep kconfig)
+	$(add_frameworks_dep kconfigwidgets)
+	$(add_frameworks_dep kcoreaddons)
+	$(add_frameworks_dep kdelibs4support)
+	$(add_frameworks_dep ki18n)
+	$(add_frameworks_dep kio)
+	$(add_frameworks_dep kitemviews)
+	$(add_frameworks_dep khtml)
+	$(add_frameworks_dep knewstuff)
+	$(add_frameworks_dep kparts)
+	$(add_frameworks_dep kplotting)
+	$(add_frameworks_dep ktextwidgets)
+	$(add_frameworks_dep kwidgetsaddons)
+	$(add_frameworks_dep kunitconversion)
+	$(add_frameworks_dep kxmlgui)
+	$(add_frameworks_dep solid)
+	$(add_kdeapps_dep libkdegames)
+	$(add_qt_dep qtdeclarative)
+	$(add_qt_dep qtgui)
+	$(add_qt_dep qtsvg)
+	$(add_qt_dep qtopengl)
+	$(add_qt_dep qtwidgets)
+	$(add_qt_dep qtxml)
+	editor? (
+		dev-cpp/eigen:3
+		sci-chemistry/avogadro
+		sci-chemistry/openbabel
+	)
+	solver? ( dev-ml/facile[ocamlopt] )
+"
+RDEPEND="${DEPEND}
+	sci-chemistry/chemical-mime-data
+"
+
+src_configure(){
+	# Fix missing finite()
+	[[ ${CHOST} == *-solaris* ]] && append-cppflags -DHAVE_IEEEFP_H
+
+	local mycmakeargs=(
+		-DCMAKE_DISABLE_FIND_PACKAGE_Eigen3=$(usex editor)
+		-DCMAKE_DISABLE_FIND_PACKAGE_AvogadroLibs=$(usex editor)
+		-DCMAKE_DISABLE_FIND_PACKAGE_OpenBabel2=$(usex editor)
+		-DCMAKE_DISABLE_FIND_PACKAGE_OCaml=$(usex solver)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Libfacile=$(usex solver)
```

Comment:
It is more readable, that is the reason why it was not removed with EAPI 6. We should not mix ebuild style in kde project.